policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Property Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.property_tax_credits
+        income = tax_unit("adjusted_gross_income", period)
+
+        age = person("age", period)
+        age_qualifies = age >= p.min_age
```

Comment:
also check for "is_tax_unit_head"

Reply:
also add the ["(b) Lived with a spouse or one or more persons and the combined income from all sources in the taxable year of all persons residing in the residence was less than five thousand five hundred one dollars."](https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01072.htm)